src/index.ts

```diff
@@ -54,5 +54,5 @@ tryCatch(() => emailService.verifyTransport(), toError)
         process.exit(1);
       })
   )
-  // tslint:disable-next-line:no-floating-promises
+  // eslint-disable-next-line @typescript-eslint/no-floating-promises
```

Comment:
This line should be moved at the top of `tryCatch` chain to work properly.

Reply:
Done.